runtime/op/switcher/ztests/switch-chained.yaml

```
@@ -0,0 +1,23 @@
+zed: |
+  switch foo (
+    case "Ex" => foo := 5
+    case "Gd" => foo := 4
+    case "TA" => foo := 3
+    case "Fa" => foo := 2
+    case "Po" => foo := 1
+    default => foo := 0
+  )
+  | switch bar (
+    case "Ex" => bar := 5
+    case "Gd" => bar := 4
+    case "TA" => bar := 3
+    case "Fa" => bar := 2
+    case "Po" => bar := 1
+    default => bar := 0
+  )
+
+input: |
+  {foo:"Gd",bar:"Fa"}
+
+output: |
+  {foo:4,bar:2}
```

Nit: Simplify program and input.

Not nit: Test both switch forms since you modified both compileExprSwitch and compileSwitch.

```suggestion
zed: |
  switch this (
    case 0 => pass
    case 1 => yield 2
  )
  | switch this (
    case 0 => pass
    case 2 => yield 3
  )
  | switch (
    case this==0 => pass
    case this==3 => yield 4
  )

input: |
  1

output: |
  4
```

---

Clever!